components/Modal/modal.js

```diff
@@ -0,0 +1,32 @@
+const simpleModalWrapper = document.querySelector("#simple-modal-wrapper");
+const simpleModalDemoBtn = document.querySelector("#simple-modal-demo-btn");
+const listModalWrapper = document.querySelector("#list-modal-wrapper");
+const listModalDemoBtn = document.querySelector("#list-modal-demo-btn");
+const actionBtns = document.querySelectorAll(".btn-action");
+
+const body = document.querySelector("body");
+
+simpleModalDemoBtn.addEventListener("click", () => {
+  simpleModalWrapper.style.display = "flex";
+});
+
+listModalDemoBtn.addEventListener("click", () => {
+  listModalWrapper.style.display = "flex";
+});
+actionBtns.forEach(btn => {
+  btn.addEventListener("click", () => {
+    if (btn.parentNode.id === "simple-modal-buttons") {
```

Comment:
You can add comments here for better code readability.

Reply:
done